components/header.js

```diff
@@ -1,14 +1,10 @@
 import Link from 'next/link';
 import { Container } from './container';
 
-export const Header = () => {
+export const Header = ({ menu }) => {
   return (
     <div className="header">
-      <Container>
-        <Link href="/">Home</Link>
-        <Link href="/projects">Projects</Link>
-        <Link href="/about-me">About Me</Link>
-      </Container>
+      <Container menu={menu}></Container>
```

Comment:
Similarly you don't need to send menu to container but you need to use menu to generate Link inside the container block
```javascript
<Container>
{
    menu.map(menuItem => <Link key={menuItem.id} href="menuItem.href">{menuItem.label}</Link>)
}
</Container>
```